packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1282,35 +1283,39 @@ void main() {
     await checkDragHandleAndColors();
   });
 
-  testWidgets('showModalBottomSheet does not use root Navigator by default', (WidgetTester tester) async {
-    await tester.pumpWidget(MaterialApp(
-      home: Scaffold(
-        body: Navigator(onGenerateRoute: (RouteSettings settings) => MaterialPageRoute<void>(builder: (_) {
-          return const _TestPage();
-        })),
-        bottomNavigationBar: BottomNavigationBar(
-          items: const <BottomNavigationBarItem>[
-            BottomNavigationBarItem(
-              icon: Icon(Icons.ac_unit),
-              label: 'Item 1',
-            ),
-            BottomNavigationBarItem(
-              icon: Icon(Icons.style),
-              label: 'Item 2',
-            ),
-          ],
+  testWidgets('showModalBottomSheet does not use root Navigator by default',
```

Comment:
Nit: formatting.
When just header is changed, do not indent the entire body of the test. 

It will keep the code concise and make code review easier. This practice is specific for code of flutter framework and does not apply to dart projects in general. 

Example: https://github.com/flutter/flutter/blob/0099d1a96ac80babc5b85cc7c7ca6764091b75f3/packages/flutter/test/widgets/image_headers_test.dart#L20

Rules: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting

Reply:
@polina-c [Revert "test: Update the tests"](https://github.com/flutter/flutter/pull/146748/commits/a6c46d4e68e35fb7cc3a60edd4e025a8e581ba2c) should have fixed it